README.md

```diff
@@ -116,6 +122,8 @@ Besides the component itself, the module also includes a `getDocument` function
 
 [Advanced Usage Demo (StackBlitz)](https://stackblitz.com/fork/vue-pdf-embed)
 
+[Public methods Usage Demo (JSFiddle)](https://jsfiddle.net/yngrdyn/eq73xyar/24/)
+
```

Comment:
Please remove this lines as this fiddle is not maintainable by the repo maintainers. Though I can reuse this and add to the Advanced Usage Demo afterwards.

Reply:
Please add this to `Advanced Usage Demo` fiddle. I think it adds value